### PR TITLE
fix: Put projects in the correct DOM order

### PR DESCRIPTION
**Pull request by ju-nc**
Previously, the order of the projects was simply the order GitHub's `repo` API call would return. There was some code that sorted based on stars that wasn't being used. 

This PR updates the code to use the sorting logic, as well as placing the projects in the correct order in the DOM so that keyboard navigators don't get super confused

Related issues: https://github.com/tophat/opensource.tophat.com/issues/63

## AuthorQA
- ✅ Verify projects are sorted by stars, descending
- ✅ Verify tab order is correct; ✅ verify screenreader navigation is correct

## Screenshots
Screen Shot 2020-12-17 at 12 19 51 PM

### CODE REVIEW

File: static/js/slack.js [hunk 1]
```diff
@@ -95,5 +95,9 @@ const handleFormSubmit = (e) => {
 
 const form = document.getElementById('slack-invite')
 const submitButton = document.getElementById('slack-submit-button')
-form.addEventListener('submit', handleFormSubmit)
-submitButton.addEventListener('click', handleFormSubmit)
+if (form) {
+    form.addEventListener('submit', handleFormSubmit)
```
Comment (ju-nc) [inline static/js/slack.js:99]: These changes are unrelated to the a11y stuff, but I noticed a console warning if the slack invite was excluded.

File: static/css/styles.css [hunk 2]
```diff
@@ -17,6 +17,16 @@ h3 {
 h2 {
     font-weight: 500;
 }
+.project-title {
+    /* h3 styling */
```
Comment (ju-nc) [inline static/css/styles.css:21]: unrelated to a11y changes, but the change from h3 -> h2 didn't look very good on close inspection

**Comment by zusyed** [comment]
Are you able to navigate through the projects using keyboard? When I press `tab`, it just scrolls all the way down.

**Comment by ju-nc** [comment]
> When I press tab, it just scrolls all the way down.

That's strange, it works on my end. What browser are you using?

**Comment by zusyed** [comment]
I am using Firefox

**Comment by zusyed** [comment]
Keyboard navigation works fine on Chrome though 🤔

**Comment by ju-nc** [comment]
If I recall, Firefox has some unique settings when it comes to tab ordering on MacOS. 

https://developer.mozilla.org/en-US/docs/Mozilla/Preferences/Preference_reference/accessibility.tabfocus

I had set this value to "7" on my settings

**Comment by zusyed** [comment]
Tested on Chrome and the tab ordering looks good. The only thing I noticed is that you need to tab twice on "View Our Open Positions" before it goes to the social media links.

**Review by zusyed — APPROVED** [review]
LGTM

**Pull request merged**